Add sample code for laravel developer (#7)
ktorio/ktor-for-xyz: laravel/README.md

}
```

> Here comes a tip to store package version number separately. We declared `*_version` variables in `build.gradle.kts`, then use the same variable name and version number pair in `gradle.properties`. Here is how it looks like:

Reviewer: ```suggestion
When working with packages, we need to declare `*_version` variables in `build.gradle.kts`, then use the same variable name and version number pair in `gradle.properties`. Here is how that looks like:
```
